src/workato_platform_cli/cli/utils/config/manager.py

```diff
@@ -251,9 +267,45 @@ async def _setup_profile(self) -> str:
         """Setup or select profile"""
         click.echo("📋 Step 1: Configure profile")
 
-        existing_profiles = self.profile_manager.list_profiles()
+        # Check for environment variables FIRST
+        env_token = os.environ.get("WORKATO_API_TOKEN")
+        env_host = os.environ.get("WORKATO_HOST")
+
         profile_name: str | None = None
 
+        if env_token or env_host:
+            # Show what was detected
+            if env_token:
+                click.echo("✓ Found WORKATO_API_TOKEN in environment")
+            if env_host:
+                region_info = self._match_host_to_region(env_host)
+                click.echo(f"✓ Found WORKATO_HOST in environment ({region_info.name})")
+
+            # Ask user if they want to use env vars
+            use_env_vars = click.confirm(
+                "Use environment variables for authentication?",
+                default=True,
+            )
+
+            if use_env_vars:
```

Comment:
When env vars are detected and user chooses to use them, they're asked to select a profile name. If they select an existing profile, the code overwrites it with env var credentials without warning:

Either:
  - Warn user: "This will overwrite the existing profile '{name}' with environment variable credentials. Continue?"
  - Or prevent selecting existing profiles when using env vars

Reply:
Good catch! It  now shows a warning and confirmation prompt before overwriting existing profiles with env var credentials.